include/cute/arch/copy_xe.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <cute/config.hpp>
+#include <cute/arch/copy.hpp>
+#include <cute/util/sycl_vec.hpp>
+
+#ifdef __SYCL_DEVICE_ONLY__
+#define SYCL_DEVICE_BUILTIN(x) SYCL_EXTERNAL extern "C" x
+#else
+#define SYCL_DEVICE_BUILTIN(x) \
+    inline x { assert(false); }
+#endif
```

Comment:
Would it be possible to move this built-ins to `include/cute/config.hpp` file

Reply:
Happy to move it. But this is intended as a temporary hack. The planned proper solution is that we will use the SPIRV/SYCL extension. And then neither this nor `SYCL_DEVICE_OCL` will be needed.

Reply:
Yes that is find, in this case lets keep it as is